src/Modules/OrchardCore.Commerce/Migrations/ProductMigrations.cs

```diff
@@ -21,6 +23,10 @@ public int Create()
     {
         _contentDefinitionManager
             .AlterPartDefinition(nameof(ProductPart), builder => builder
+                .WithField("ProductImage", field => field
```

Comment:
Since OCC is already releases we should use UpdateFromN when editing existing migrations.